build.ps1

```diff
@@ -1,12 +1,20 @@
+$SCRIPT_ROOT = "$PSScriptRoot"
+$SdkVersion = Get-Content (Join-Path $SCRIPT_ROOT "DotnetCLIVersion.txt")
+
 $env:DOTNET_CLI_TELEMETRY_OPTOUT = 1
 $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 1
+$env:NUGET_PACKAGES = "$ScriptRoot\packages\"
+
+& "$SCRIPT_ROOT\init-tools.cmd"
+
+$CLIPATH = "$SCRIPT_ROOT\Tools\dotnetcli"
+$SDKPATH = "$CLIPATH\sdk\$SdkVersion"
 
-$ScriptRoot = "$PSScriptRoot"
-$SdkVersion = Get-Content (Join-Path $ScriptRoot ".cliversion")
+& "$CLIPATH\dotnet" restore "$SCRIPT_ROOT\tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj"
+& "$CLIPATH\dotnet" build "$SCRIPT_ROOT\tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj"
 
-& "$ScriptRoot\bootstrap\bootstrap.ps1"
-& "$ScriptRoot\Tools\dotnetcli\dotnet" restore "$ScriptRoot\tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj"
-& "$ScriptRoot\Tools\dotnetcli\dotnet" build "$ScriptRoot\tasks\Microsoft.DotNet.SourceBuild.Tasks\Microsoft.DotNet.SourceBuild.Tasks.csproj"
-& "$ScriptRoot\Tools\dotnetcli\dotnet" "$ScriptRoot\Tools\dotnetcli\sdk\$SdkVersion\MSBuild.dll" "$ScriptRoot\build.proj" @args /t:WriteDynamicPropsToStaticPropsFiles /p:GeneratingStaticPropertiesFile=true
-& "$ScriptRoot\Tools\dotnetcli\dotnet" "$ScriptRoot\Tools\dotnetcli\sdk\$SdkVersion\MSBuild.dll" "$ScriptRoot\build.proj" @args
+Remove-Item -Recurse -Force "$NUGET_PACKAGES"
 
+& "$CLIPATH\dotnet" "$SDKPATH/MSBuild.dll" "$SCRIPT_ROOT/build.proj" "$@" /t:WriteDynamicPropsToStaticPropsFiles /p:GeneratingStaticPropertiesFile=true
+& "$CLIPATH\dotnet" "$SDKPATH/MSBuild.dll" "$SCRIPT_ROOT/build.proj" "$@" /t:GenerateRootFs
```

Comment:
We don't need to call GenerateRootFs on windows.

Reply:
Removed.